### ISSUE TITLE
[1.12.2] Invisible mobs in the Twilight Forest

### ISSUE THREAD

**Issue by Zsashas** [Lycanite/LycanitesMobs#286]
Recently, every time I go to the Twilight Forest I am beset by a few invisible Wargs and/or Calpods and/or Ents (it's hard to tell). Not even The One Probe can detect them. The only way I know is because I'm getting hit, and because the wargs are so loud, and the occasional warg slowness) Randomly throwing TiCon shurikens usually deals with it but it's getting annoying. I can see Trents in the distance so it isn't all mobs in the TF.

**Comment by Lycanite** [comment]
Hmm that's very strange, I'm not sure how they're doing that, I'll have to run some tests on it. Could there be any other mods involved?

**Comment by Zsashas** [comment]
Possibly, I do have a sizeable pack at this point. It doesn't affect all of them, though, just the few that seem to spawn around me after entering the TF, and not even every time.

https://hastebin.com/ayohuzekig.css

**Comment by Lycanite** [comment]
Yeah that looks like a lot of mods, I'd try running with just Lycanites Mobs and Twilight Forest in a test world first and then slowly add groups of mods back in until the problem returns. This problem doesn't seem to be happening in packs like the Madpack 4 (on JonBams stream) so far, but I'll run some tests of my own to be sure when I get the time and update my own pack too.

**Comment by Lycanite** [comment]
I've ran some tests with my own pack and I can't replicate this issue so it could be other mods, etc.

**Comment by Draylynn** [comment]
Similar issue in my pack, only it appears to be pigs/villagers and a few random aggressive overworld mobs, started happening after Lycanites Mobs were added.
[ModList.txt](https://github.com/Lycanite/LycanitesMobs/files/1613748/ModList.txt)

**Comment by Lycanite** [comment]
Hmm strange, I'll get my own modpack updated which uses TF and will do some testing.
